app/src/main/java/samyups/example/trainingss/data/Dog.kt

```diff
@@ -1,7 +1,11 @@
 package samyups.example.trainingss.data
 
+import androidx.room.Entity
+import androidx.room.PrimaryKey
+
+@Entity(tableName = "Dog Breeds")
```

Comment:
I would avoid putting a space in the name. Then you have to do weird things like use a ` symbol to let the system know there is a space in the name. It's better to call it `Dog_Breeds` or `DogBreeds`. There is probably a recommendation on how to name tables probably that you can look up.